src/index.css

```diff
@@ -1,5 +1,123 @@
 body {
+  background: #fff5fe;
   margin: 0;
   padding: 0;
   font-family: sans-serif;
 }
+
+@keyframes beat{
+	to { transform: scale(1.05); }
+}
+
+svg {
+  overflow: inherit !important;
+}
+
+#organ_5, #organ_10, #organ_11, #organ_17, #organ_16 {
+  cursor: pointer;
+  fill-opacity: 0;
+  stroke-opacity: 0.2;
+  vector-effect: "non-scaling-stroke";
+  fill: #FF0000;
+  stroke: #FF0000;
+}
+
+#organ_5 {
+  animation: beat 1.3s infinite alternate;
+  fill-opacity: 0.6;
+}
+
+#pression-disable {
+  stroke: #004eff;
+  fill: transparent;
+  vector-effect: "non-scaling-stroke";
+  stroke-width: 5;
+  fill-opacity: 0.6;
+  stroke-opacity: 0.2;
+  cursor: pointer;
+}
+
+#organ_11, #cord {
+  cursor: pointer;
+  fill: rgb(245 181 173);
+  fill-opacity: 1;
+}
+
+ #cord:hover, #organ_17:hover, #organ_5:hover, #organ_10:hover, #organ_11:hover, #organ_17:hover, #organ_16:hover {
+  fill: #FF0000;
+  fill-opacity: 0.8;
+  stroke-opacity: 1;
+}
+
+#pression-disable:hover {
+  fill-opacity: 0.8;
+  stroke-opacity: 1;
+}
+
+.heart-enable {
+  animation: beat .3s infinite alternate !important;
+  fill-opacity: 0.8 !important;
+  stroke-opacity: 1 !important;
+}
+
+.pression-enable-nerf-bottom {
+  stroke: rgb(200, 159, 64) !important;
+}
+
+#nerfs {
+  stroke-width: 2;
+}
+
+#Nerf-bottom, #Nerf-mid, #Nerf-top {
+  cursor: pointer;
+  stroke: rgb(234, 211, 193);
+  stroke-linecap: square;
+}
+
+#Nerf-bottom:hover, #Nerf-mid:hover, #Nerf-top:hover {
+  stroke: rgb(200, 159, 64);
+}
+
+#Bulbe-tige {
+  cursor: pointer;
+  fill: rgb(254, 202, 156);
+}
+
+#Bulbe-tige:hover {
+  fill: rgb(224, 141, 67);
+}
+
+#Bulbe-node {
+  cursor: pointer;
+  fill: rgb(203, 152, 106);
+}
+
+#Bulbe-node:hover {
+  fill: rgb(253, 134, 26);
+}
+
+#Bulbe-left, #Bulbe-right {
+  cursor: pointer;
+  fill: rgb(181, 123, 241);
+}
+
+#Bulbe-left:hover, #Bulbe-right:hover {
+  fill: rgb(203, 156, 252);
+}
+
+#Bulbe-node-left, #Bulbe-node-right {
+  cursor: pointer;
+  fill: rgb(151, 103, 152);
+}
+
+#Bulbe-node-left:hover, #Bulbe-node-right:hover {
+  fill: rgb(141, 21, 144);
+}
+
+#Oval_1, #Oval_2, #Line, #Line-2 {
+  cursor: pointer;
+}
+
+.sices-enable {
```

Comment:
What is `sices`? Why do you use `-` here and `_` above? You need to be consistent.